src/main/java/it/pagopa/interop/probing/caller/annotations/ValidateStringArraySize.java

```diff
@@ -0,0 +1,24 @@
+
+package it.pagopa.interop.probing.caller.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import it.pagopa.interop.probing.caller.annotations.validator.StringArrayValidator;
+
+@Target({ ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = StringArrayValidator.class)
+public @interface ValidateStringArraySize {
+
+	int maxSize();
+
+	String message() default "One of the strings of the array is more than {maxSize} characters long";
```

Comment:
Prepare for Internazionalization i18n of messages

Reply:
for now, internationalisation has only been planned for the FE side, but we can discuss it for the BE side